### PR TITLE
New version: Huawei.QuickAppIde version 14.0.1

### PR DESCRIPTION
**Pull request by SpecterShell**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146914)

### CODE REVIEW

File: manifests/h/Huawei/QuickAppIde/14.0.1/Huawei.QuickAppIde.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+FileExtensions:
+- card
+- ux
+ReleaseDate: 2024-03-06
+Installers:
+- Architecture: x64
+  InstallerUrl: https://contentcenter-vali-drcn.dbankcdn.cn/pvt_2/DeveloperAlliance_package_901_9/5e/v3/aFdD1fSZSOOrRtqsWG14Pw/QuickAppIDE-V14.0.1-Win64.exe?HW-CC-KV=V1&HW-CC-Date=20240306T073107Z&HW-CC-Expire=315360000&HW-CC-Sign=0422592C5622FEA1EAE520EDEDBBBB2BBEB95053B96178BC1FB66481C85174E9
```
Comment (stephengillie) [inline manifests/h/Huawei/QuickAppIde/14.0.1/Huawei.QuickAppIde.installer.yaml:15]:
```suggestion
  InstallerUrl: https://appfile-cn.dbankcdn.com/FileServer/getFile/appAttach/011/111/111/0000000000011111111.20221114162359.00120356616353505553714097760392:20471231000000:0001:0A825835C422CD1C421C0D46615896E8A6339C6BECCA31651023273FBC514B34.exe?needInitFileName=true
```

**Review by stephengillie — COMMENTED** [review]
Hi @SpecterShell,

I could only find up to version `12.7` in the `PackageUrl`, and the `InstallerUrl` was different. Where did you find this version URL?

File: manifests/h/Huawei/QuickAppIde/14.0.1/Huawei.QuickAppIde.installer.yaml (hunk 1 above)
Comment (SpecterShell) [reply manifests/h/Huawei/QuickAppIde/14.0.1/Huawei.QuickAppIde.installer.yaml:15]: The Chinese site. The English one is a bit outdated.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**